src/bindings/fs.rs

```diff
@@ -8,6 +8,13 @@ pub fn init(lua: &Lua) -> Result<(), LuaError> {
 
     let module = lua.create_table()?;
 
+    module.set("canonicalize", lua.create_function( |lua, path: String| {
+        match fs::canonicalize(path).map_err(|err| LuaError::external(err)) {
+            Ok(i) => Ok(Some(lua.create_string(&i.to_str().unwrap()).unwrap())),
```

Comment:
Try to avoid unwrapping when possible

Reply:
Sure, that's my first ever code in Rust, so I don't exactly understand how to escape those two calls.